src/mainboard/seeed/beaglev/src/main.rs

```diff
@@ -0,0 +1,122 @@
+#![feature(llvm_asm)]
```

Comment:
Do you actually need this?

Reply:
yeah. there's an include. Well, we used to need it, do we not need it?